slither.config.json

```diff
@@ -0,0 +1,5 @@
+{
+  "detectors_to_exclude": "external-function",
```

Comment:
Here is the documentation behind this detector:
https://github.com/crytic/slither/wiki/Detector-Documentation#public-function-that-could-be-declared-external

There is opinion whether this is useful anymore with the latest compiler.